src/main/resources/assets/createframed/lang/ru-ru.json

```
@@ -0,0 +1,217 @@
+{
+  "itemGroup.createframed.create_framed": "Create: Framed",
+
+  "block.createframed.copper_window": "Медное окно",
+  "block.createframed.copper_window_pane": "Медная оконная панель",
+  "block.createframed.zinc_window": "Цинковое окно",
+  "block.createframed.zinc_window_pane": "Цинковая оконная панель",
+  "block.createframed.brass_window": "Латунное окно",
+  "block.createframed.brass_window_pane": "Латунная оконная панель",
+  "block.createframed.rose_quartz_window": "Окно из розового кварца",
+  "block.createframed.rose_quartz_window_pane": "Оконная панель из розового кварца",
+  "block.createframed.andesite_alloy_window": "Окно из андезитного сплава",
```

For consistency with Create translation this line and the line below should have "андезитового", not "андезитного":
```suggestion
  "block.createframed.andesite_alloy_window": "Окно из андезитового сплава",
```